projects/plugins/crm/admin/settings/tax.page.php

```diff
@@ -138,7 +138,26 @@
 					<div class="ui info icon message">
 						<div class="content">
 							<div class="header"><?php esc_html_e( 'No Tax Rates', 'zero-bs-crm' ); ?></div>
-							<p><?php echo wp_kses( sprintf( __( 'There are no tax rates defined yet. Do you want to <a href="%s" id="zbs-new-add-tax-rate">create one</a>?', 'zero-bs-crm' ), '#' ), $zbs->acceptable_restricted_html ); ?></p>
+							<p>
+								<?php
+									echo sprintf(
+										wp_kses(
+											/* Translators: placeholder is an anchor, which gets changed via Javascript elsewhere to create new tax rate entry fields. */
+											__(
+												'There are no tax rates defined yet. Do you want to <a href="%s" id="zbs-new-add-tax-rate">create one</a>?',
+												'zero-bs-crm'
+											),
+											array(
+												'a' => array(
+													'href' => array(),
+													'id'   => array(),
+												),
+											)
```

Comment:
Would it make sense to continue to use `$zbs->acceptable_restricted_html` for consistency and then add `id => array()` to the `ZeroBSCRM::$acceptable_restricted_html` ruleset?

IMO allowing IDs in comments should, generally, be OK?

Reply:
Ah perfect, thanks!